walrus/api.py

```diff
@@ -1,28 +1,31 @@
-import os.path
-import pandas as pd
+import os
+from walrus.FileConfig import FileConfig
 
 file_dir = os.path.dirname(os.path.realpath(__file__))
 file_path = os.path.join(file_dir, 'data', 'country-capitals.csv')
+file_object = FileConfig(file_path)
```

Comment:
object names should be variant of ClassName. For example, `file_config` if the class is `FileConfig, `country` for an object of type `Country` etc.

Reply:
Appended this accordingly too.